api/source/service/mysql/STIGService.js

```diff
@@ -324,6 +319,22 @@ exports.queryRules = async function ( ruleId, inProjection ) {
   
   // PROJECTIONS
   // Include extra columns for Rules with details OR individual Rule
+  if ( inProjection && inProjection.includes('detail') ) {
+    columns.push(
```

Comment:
Please define a groupBy array that you can push the same column to

Reply:
Added